src/cdk/explorer/tree/treeInspector.ts

```diff
@@ -63,3 +63,18 @@ export function getDisplayLabel(construct: ConstructTreeEntity): string {
 
     return construct.id
 }
+
+/** 
+* @param construct CDK construct
+*/
+export function isStateMachine(construct: ConstructTreeEntity): boolean {
+
+    if (construct.children) {
```

Comment:
Instead of wrapping the main logic of this function in an if block, we can make the function easier to follow by changing 
```
if (construct.children) {
 ...do stuff...
} 
return false
``` 

to 

```
if (!construct.children) {
  return false
}
... do stuff ...
```